gnovm/tests/files/var18.gno

```diff
@@ -0,0 +1,8 @@
+package main
+
+func main() {
+	a, b, c := 1, 2
+}
+
+// Error:
+// main/files/var18.gno:4: should not happen
```

Comment:
let's change this to be  something descriptive, thank you.

Reply:
Yes I will do it this PR: #2206

Maybe I should remove this example, in this PR ?

Reply:
OK. I think you can leave it as a show case of  `correct loc` info.